$OpenBSD$

 * trick ocamlfind into not updating systemwide ld.conf at fake time
 * missing linker flags

--- src/OCamlMakefile.orig	Fri Jul  2 16:23:36 2010
+++ src/OCamlMakefile	Thu Apr 19 19:41:23 2012
@@ -86,6 +86,7 @@ export IDLFLAGS
 export OCAMLDOCFLAGS
 
 export OCAMLFIND_INSTFLAGS
+OCAMLFIND_INSTFLAGS += -ldconf ld.conf.dummy
 
 export DVIPSFLAGS
 
@@ -214,7 +215,7 @@ endif
 export OCAMLYACC
 
 ifndef OCAMLMKLIB
-  OCAMLMKLIB := ocamlmklib
+  OCAMLMKLIB := ocamlmklib ${LDFLAGS}
 endif
 export OCAMLMKLIB
 
@@ -678,7 +679,7 @@ endif
 # USER RULES
 
 # Call "OCamlMakefile QUIET=" to get rid of all of the @'s.
-QUIET=@
+QUIET=
 
 # generates byte-code (default)
 byte-code:		$(PRE_TARGETS)
